src/jit/lower.cpp

```diff
@@ -104,6 +104,34 @@ bool Lowering::IsSafeToContainMem(GenTree* parentNode, GenTree* childNode)
 }
 
 //------------------------------------------------------------------------
+// IsContainableMemoryOp: Checks whether this is a memory op that can be contained.
+//
+// Arguments:
+//    node - the node of interest.
+//
+// Notes:
+//    This differs from the isMemoryOp() method on GenTree because it checks for
+//    the case of an untracked local. Note that this won't include locals that
+//    for some reason do not become register candidates, nor those that get
+//    spilled.
+//
+// Return value:
+//    True if this will definitely be a memory reference that could be contained.
+//
+bool Lowering::IsContainableMemoryOp(GenTree* node)
+{
+    if (node->isMemoryOp())
+    {
+        return true;
+    }
+    if (!node->IsLocal())
+    {
```

Comment:
Should this be written this way instead?
(To me it reads better)
```
if (node->IsLocal())
{
   // When we have a Local Var or Fld, return true for non-tracked variables, since they always live on the stack
   return !comp->lvaTable[node->AsLclVar()->gtLclNum].lvTracked;
}
return false;
```